daemon/cmd/kube_proxy_replacement.go

```diff
@@ -149,7 +149,11 @@ func initKubeProxyReplacementOptions(sysctl sysctl.Sysctl, tunnelConfig tunnel.C
 			log.Warning("NodePort BPF configured without bind(2) protection against service ports")
 		}
 
-		if option.Config.TunnelingEnabled() && tunnelConfig.Protocol() == tunnel.VXLAN &&
+		if option.Config.TunnelingEnabled() && tunnelConfig.UnderlayProtocol() == tunnel.IPv6 {
+			return fmt.Errorf("BPF NodePort cannot be used over an IPv6 underlay")
```

Comment:
and here

Reply:
Support for this is actually already ready (see follow-up PR mentioned above) and I'm just splitting the work.